pkgs/by-name/br/braa/package.nix

```
@@ -0,0 +1,29 @@
+{
+  lib,
+  stdenv,
+  fetchurl,
+  zlib,
+}:
+stdenv.mkDerivation rec {
+  pname = "braa";
+  version = "0.82";
+
+  src = fetchurl {
+    url = "http://s-tech.elsat.net.pl/${pname}/${pname}-${version}.tar.gz";
+    sha256 = "d650d34cdf30fcbbbbf4e4f11c8f5d51fd5caa3cca58d620fa1d978b8c18919c";
+  };
```

I believe the correct fetcher here would be `fetchzip`

```suggestion
  src = fetchzip {
    url = "http://s-tech.elsat.net.pl/${pname}/${pname}-${version}.tar.gz";
    hash = "d650d34cdf30fcbbbbf4e4f11c8f5d51fd5caa3cca58d620fa1d978b8c18919c";
  };
```

---

@h7x4 that hash would be incorrect regardless fyi